src/points.js

```diff
@@ -12,6 +12,7 @@ module.exports.write = function writePoints(coordinates, extent, shpView, shxVie
         // 4 record number
         // 4 content length in 16-bit words (20/2)
         shpView.setInt32(shpI, i);
+        shpView.setInt32(shpI, i + 1);
```

Comment:
could you add an explanation of what this is

Reply:
Sure! Record numbers in shp file headers are actually 1-indexed, not 0-indexed. You can read the technical spec here and the reference to the record header is on page 9 (https://www.esri.com/content/dam/esrisites/sitecore-archive/Files/Pdfs/library/whitepapers/pdfs/shapefile.pdf) 

![image](https://github.com/Agriculture-Intelligence/shp-write/assets/1749334/d7901d3e-c6c9-4ad9-aeeb-a9004801d043)

You can also see that the `writePoints` method for `poly.js` already did this correctly

https://github.com/Agriculture-Intelligence/shp-write/blob/master/src/poly.js#L29